docs/evaluate/development-production-features/schedules.mdx

```diff
@@ -0,0 +1,47 @@
+---
+id: schedules
+title: Schedules - Temporal feature
+description: Learn the benefits of scheduling Temporal Workflows and explore best practices to ensure timely and efficient execution of your business processes.
+sidebar_label: Schedules
+tags:
+  - schedules
+  - Workflow-automation
+  - temporal-Workflows
+  - scheduling
+keywords:
+  - temporal schedules
+  - automate Workflows
+  - schedule Workflow executions
+  - temporal scheduling best practices
+  - Workflow automation temporal
+  - schedule management
+  - temporal Workflow scheduling
+---
+
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
+
+Temporal Schedules is a feature that allows you to "schedule" Temporal Workflows at specified times or intervals, adjusting for peak use.
+
+It offers a flexible way to automate and manage your Temporal Workflows, ensuring your business processes run smoothly and efficiently, especially when handling time-sensitive tasks.
+
+1. **Automate Repetitive Tasks:** 
+   Schedules automate repetitive tasks, reducing manual intervention and ensuring timely execution of business processes.
+2. **Enhanced Workflow Control and Observability:** 
+   Gain complete control over your automation processes. With Schedules, you can create, backfill, delete, describe, list, pause, trigger, and update Workflow Executions.
+3. **Flexible Timing:** 
+   Schedule Workflow Executions to run at regular intervals or specific future times, ensuring they execute precisely when needed.
+4. **Reliable and Scalable:** 
+   Designed for reliability and scalability, Temporal Schedules handle the complexities of distributed systems while ensuring your Workflows run as intended, even during failures.
+5. **Eliminate External Dependencies:** 
+   Schedules remove the need to integrate external scheduling systems.
+
+Jump straight to a Temporal SDK feature guide.
```

Comment:
I don't care for this CTA and feel it could be stronger. Instead of telling them to "go to this section to learn more" just say "Use this feature". I'm not certain my wording is best, but I think the idea is there. 

```suggestion
Use schedules with the Temporal SDK.
```

Reply:
I think I might keep the CTA for now since all the other pages have it. But I'm starting to add blog posts, webinars, etc to that list so it will need to end up being changed since it's not just a feature guide anymore. I'll have to think of a better phrase too!

Reply:
Sounds good.